src/main/com/wsscode/pathom3/connect/planner.cljc

```diff
@@ -910,7 +914,8 @@
                         (assoc
                           ::attr-resolvers-trail (into (or attr-resolvers-trail #{}) resolvers)
                           ::available-data available
-                          :edn-query-language.ast/node (pfsd/shape-descriptor->ast missing))))]
+                          :edn-query-language.ast/node (pfsd/shape-descriptor->ast missing)
+                          :com.wsscode.pathom3.error/lenient-mode? lenient-mode?)))]
```

Comment:
For `shape-reachable?` we can always use lenient mode `true`

Reply:
Cool, that makes sense though I think that changes current behaviour if I remember correctly. I'll double check and get back to you